docs/source/reference/kubernetes/index.rst

```diff
@@ -82,7 +91,7 @@ Once your cluster administrator has :ref:`setup a Kubernetes cluster <kubernetes
 
    .. code-block:: console
 
-        $ sky launch --cpus 2+
```

Comment:
nit: is task.yaml optional?

Reply:
Yes, it's optional - `sky launch --cpus 2+` also works. However, I added task.yaml to avoid confusing users ("Where do I specify my task yaml? Is Kubernetes any different from regular SkyPilot in this regard?").